src/OSPSuite.UI/Extensions/UxChartControlExtensions.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using DevExpress.Utils;
+using DevExpress.XtraCharts;
+using OSPSuite.Core.Chart;
+using OSPSuite.Core.Domain;
+using OSPSuite.Presentation.Extensions;
+using OSPSuite.UI.Controls;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.UI.Extensions
+{
+   public static class UxChartControlExtensions
+   {
+      private const string WATERMARK_ANNOTATION = "WATERMARK_ANNOTATION";
+
+      /// <summary>
+      ///    Copies the chart to clipboard as an image using export settings if defined
+      ///    Otherwise uses current visual settings
+      /// </summary>
+      public static void CopyToClipboard(this UxChartControl chartControl, IChart chart, string watermark)
+      {
+         using (var cloneOfChartControl = (ChartControl) chartControl.Clone())
+         {
+            cloneOfChartControl.SetFontAndSizeSettings(chart.FontAndSize, chartControl.Size);
+            if (chart.IncludeOriginData)
+               AddOriginData(cloneOfChartControl, chart);
+
+            if (!string.IsNullOrEmpty(watermark))
+               AddWatermark(cloneOfChartControl, chart, watermark);
+
+            chartControl.CopyChartToClipboard(cloneOfChartControl);
+         }
+      }
+
+      public static void AddWatermark(this ChartControl chartControl, IChart chart, string watermark)
+      {
+         var shouldHideWatermark = string.IsNullOrEmpty(watermark);
+         var watermarkAnnotation = chartControl.Annotations.OfType<TextAnnotation>().FirstOrDefault(x => Equals(x.Name, WATERMARK_ANNOTATION));
+         //we add a watermark but this should be removed
+         if (watermarkAnnotation != null && shouldHideWatermark)
+            chartControl.Annotations.Remove(watermarkAnnotation);
+
+         if (shouldHideWatermark)
+            return;
+
+         if (watermarkAnnotation == null)
+            watermarkAnnotation = createWatermarkAnnotation(chartControl);
+
+         watermarkAnnotation.Font = new Font(chart.FontAndSize.Fonts.FontFamilyName, Constants.ChartFontOptions.DEFAULT_FONT_SIZE_WATERMARK);
```

Comment:
We could define a variable editable by the user here. Not sure if this should be done tho (otherwise it could become quite small)

Reply:
Would make sense I think